packages/graphql/src/queries/lists.graphql

```diff
@@ -1,4 +1,26 @@
-# predicateId = 4 for tags
+# predicateId = 4 for tags on dev, 3 on prod
```

Comment:
Thinking about how we handle this in the future -- we can prob expose these from some sort of utils package / config package in the future. We'll handle that once everything settles